content/docs/ref/send-comment.md

```diff
@@ -0,0 +1,19 @@
+# Command Reference: `send-comment`
+
+```dvc
+cml send-comment some_content.md
+```
+
+## Common error messages
+
+### Bitbucket
+
+- Can't create a pull request comment: the Pull Request Commit Links application
+  has not been installed.
+
+  - We don't like ClickOps either but here's an actual
```

Comment:
Minor: does it have to be a nested bullet point? 🤔

Reply:
unnecessary future-proofing indeed